docs/development.md

```diff
@@ -0,0 +1,26 @@
+# Development Notes
+
+## Arrow Functions
+
+This projet heavily prefers the use of the `function` keyword over the use of 
+[arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) 
+in most cases.
+Arrow functions may be used for one line callbacks within methods such as `Array.map()` and `Array.filter()`.
+If the right hand side of the arrow function is more than a single expression,
+use the `function` keyword instead.
+
+
+Ensure that both the righthand side and lefthand side of the arrow wrapped in parentheses.
+For example:
+
+```javascript
+// Good
+numbers.map((number) => (number * 2));
+
+// Bad
+numbers.map(number => number * 2);
+numbers.map(number => {
+		return number * 2;
+});
+numbers.map((number) => number * 2);
```

Comment:
Go ahead and breakup the bad cases so that each can have its own command saying why it is bad.